profile/CSIP.xml

```diff
@@ -1233,18 +1232,19 @@
             </requirement>
             <requirement ID="CSIP106" REQLEVEL="MUST" EXAMPLES="structMapExample2">
                 <description>
-                    <head>Representation division identifier</head>
+                    <head>Representations division identifier</head>
                     <p xmlns="http://www.w3.org/1999/xhtml">Mandatory, xml:id identifier must be unique within the package.</p>
                     <dl xmlns="http://www.w3.org/1999/xhtml">
                         <dt>METS XPath</dt><dd>structMap/div/div/@ID</dd>
                         <dt>Cardinality</dt><dd>1..1</dd>
                     </dl>
                 </description>
             </requirement>
-            <requirement ID="CSIP107" REQLEVEL="MUST" RELATEDMAT="VocabularyFileGrpAndStructMapDivisionLabel" EXAMPLES="structMapExample2">
+            <requirement ID="CSIP107" REQLEVEL="MUST" RELATEDMAT="VocabularyFileGrpAndStructMapDivisionLabel CSIP64" EXAMPLES="structMapExample2">
                 <description>
-                    <head>Representation division label</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">The representation division (div) element in the package uses the path to the METS document as the value for the attribute LABEL.</p>
+                    <head>Representations division label</head>
+                    <p xmlns="http://www.w3.org/1999/xhtml">The representation division (div) element in the package uses the path to the METS document describing the representation as the value for the attribute LABEL.</p>
```

Comment:
The package's representation division (div) element @LABEL attribute value must be the path to the representation level METS document.